server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -136,7 +136,8 @@ static TransportVersion def(int id) {
     public static final TransportVersion SHARD_SIZE_PRIMARY_TERM_GEN_ADDED = def(8_500_072);
     public static final TransportVersion COMPAT_VERSIONS_MAPPING_VERSION_ADDED = def(8_500_073);
     public static final TransportVersion V_8_500_074 = def(8_500_074);
-    public static final TransportVersion NODE_INFO_INDEX_VERSION_ADDED = def(8_500_075);
+    public static final TransportVersion ML_TRAINED_MODEL_METADATA_PLATFORM_ADDED = def(8_500_075);
+    public static final TransportVersion NODE_INFO_INDEX_VERSION_ADDED = def(8_500_076);
```

Comment:
The new constant added by this PR needs to have the highest number. You can't change the number for a version somebody else added.

Reply:
I think this was due to a merge conflict. I will make sure to correct the TransportVersion in the final PR, as it is changing constantly during development.